src/mappings/PoolRegistry.ts

```diff
@@ -40,12 +40,17 @@ export function handlePoolUpdated(call: PoolUpdated): void {
   ])
   let poolId = call.params.pool.toHexString()
   let hash = call.params.data
+
+  upsertPool(poolId, hash);
+}
+
+export function upsertPool(poolId: string, hash: string) {
```

Comment:
Missing a return type here:
```suggestion
export function upsertPool(poolId: string, hash: string): void {
```